Add Deployments as Code Documentation (#1632)
astronomer/docs: astro/cli/astro-deployment-update.md

| Option                         | Description                                                                            | Possible Values                                                                |
| ------------------------------ | -------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------ |
| `--dag-deploy` |Enable or disable DAG-only deploys for the Deployment.                                                                                                                         | Either `enable` or `disable`. Contact [Astronomer support](https://cloud.astronomer.io/support) before using `disable` to disable the feature.                                                             |

> | `--deployment-file`        | Location of file containing the deployment's updated configuration. File can be in either JSON or YAML format.                                                                                                        | any YAML or JSON Deployment file             |

Reviewer: ```suggestion
| `--deployment-file`        | Location of the file containing the updated configuration file for the Deployment. The file format can be JSON or YAML.                                                                                                        | any YAML or JSON Deployment file             |
```

> | `--deployment-file`        | Location of the updated Deployment configuration file. The file format can be JSON or YAML. See [Create a Deployment with a Deployment File](manage-deployments-as-code.md#create-a-deployment-with-a-deployment-file).                                                                                                        | YAML or JSON Deployment configuration file             |

Reviewer 2: Make wording consistent here?

> | `--deployment-file`        | Location of the updated Deployment configuration file. The file format can be JSON or YAML. See [Create a Deployment with a Deployment File](manage-deployments-as-code.md#create-a-deployment-with-a-deployment-file).                                                                                                        | YAML or JSON Deployment configuration file             |

Reviewer 3: ```suggestion
| `--deployment-file`        | Location of the template file to update the Deployment with. The file format can be JSON or YAML. See [Create a Deployment with a Deployment File](manage-deployments-as-code.md#create-a-template-file-from-an-existing-deployment).                                                                                                        | YAML or JSON Deployment configuration file             |
```

> | `--deployment-file`        | Location of the template file to update the Deployment with. The file format can be JSON or YAML. See [Create a Deployment with a Deployment File](manage-deployments-as-code.md#create-a-template-file-from-an-existing-deployment).                                                                                                        | YAML or JSON Deployment configuration file             |

Reviewer 2: ```suggestion
| `--deployment-file`        | Location of the template file to update the Deployment with. The file format can be JSON or YAML. See [Create a Deployment with a Deployment File](manage-deployments-as-code.md#create-a-template-file-from-an-existing-deployment).                                                                                                        | A valid file path to any YAML or JSON Deployment configuration file             |
```

# Force update a Deployment
$ astro deployment update cl03oiq7d80402nwn7fsl3dmv -d="My Deployment Description" --force

> # Specify the updated Deployment's configuration with a yaml file

Reviewer 2: ```suggestion
# Update the Deployment according to the configurations specified in the YAML Deployment file
```

# Force update a Deployment
$ astro deployment update cl03oiq7d80402nwn7fsl3dmv -d="My Deployment Description" --force

# Specify the updated Deployment's configuration with a yaml file
$ astro deployment update --deployment-file deployment.yaml
```

> 
> 

Reviewer 3: ```suggestion
```

# Force update a Deployment
$ astro deployment update cl03oiq7d80402nwn7fsl3dmv -d="My Deployment Description" --force

# Specify the updated Deployment's configuration with a yaml file
$ astro deployment update --deployment-file deployment.yaml
```

:::note

> For more inforamtion on updating  a Deployment using a Deployment file read [Update a Deployment with a Deployment File](manage-deployments-as-code.md#update-a-deployment-with-a-deployment-file)

Reviewer: ```suggestion
To use a Deployment file to update a Deployment, see [Update a Deployment with a Deployment File](manage-deployments-as-code.md#update-a-deployment-with-a-deployment-file).
```
